package/MDAnalysis/coordinates/memory.py

```diff
@@ -181,14 +181,11 @@
    :inherited-members:
 
 """
-import logging
 import errno
 import numpy as np
 import warnings
-import copy
 
 from . import base
-from .timestep import Timestep
```

Comment:
If I remember correctly, this is an intentional import that isn't used - we import Timestep here to not break the API since we used to have Timestep defined here (if I remember correctly? it's been a while since that happened - the plan was to remove in 3.0)

Reply:
To be clear to @vachram97 : do not remove the import